src/app/routes/_profile.router.js

```diff
@@ -190,8 +190,9 @@ export default (app, router) => {
    */
   router.put('/api/profile', async (req, res) => {
     // remove readonly data
-    if (req.body.local) delete req.body.local;
+    if (req.body.local.password) delete req.body.local.password;
```

Comment:
I think this will remove the password from the user model completely later at `user.save()` :open_mouth: